cloudferrylib/utils/filters.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, filter_yaml):
         self.filter_yaml = filter_yaml
 
     @abc.abstractmethod
-    def get_filters(self):
+    def get_filter(self):
```

Comment:
This change will break cinder volume functionality. Let's keep the interface and return list of filters, it seems to be more flexible.